index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./master.css">
+    <title>Twitter. It's what's happening.</title>
+    <script src="https://kit.fontawesome.com/ec2fc2739a.js" crossorigin="anonymous"></script>
+    <link href="file:///Users/ashyamanning/Downloads/favicon.ico" rel="icon" type="image/x-icon" />
+  </head>
+  <body>
+    <!-- <img id="backgroundImage" src="./images/birdbackground.png" alt=""> -->
+    <div id="pageContent">
+    
+    <section id="blue">
+        <div id="left">
+
+            <div id="userContent">
+                <ul>
+                    <li id="li1" class="highlight">Follow your interests.</i></li>
+                    <li id="li2" class="highlight">Hear what people are talking about.</li>
+                    <li id="li3" class="highlight">Join the conversation.</li>
+                </ul>
+            </div>
+
+        </div>
+    </section>
+    <section id="white">
+            <div id="right">
+
+                <form action="">
+
+                    <div id="userSignIn">
+                        <input type="text" placeholder="Phone, email, or username">
+                        <input type="text" placeholder="Password">
+                        <button id="logIn1">Log in</button>
+                        <br>
+                        <a href="https://twitter.com/account/begin_password_reset">Forgot Password?</a>
+                    </div>
+                    <div id="userGreeting">
+                        <img id="birdIcon" src="http://www.transparentpng.com/thumb/twitter/twitter-bird-logo-pictures-0.png" alt="twitter bird logo pictures @transparentpng.com">
+                        <h1>See what's happening in the world right now</h1>
+                        <br>
+                        <h2>Join twitter today.</h2>
+                        <br>
+                        <button id="signIn" class="buttons">Sign up</button>
+                        <br>
+                        <button id="logIn2" class="buttons">Log in</button>
+                    </div>  
+
+                </form>
+
+            </div>
+        </section>
+    </div>
+    <div id="bottom">
+    <section id="footer">
```

Comment:
You don't have to have a `<section id="footer">` tag here, you are already using the tag `<footer>` right after. With such descriptive unique tags like `<footer>` you don't even need to give them an id, since that footer is the only element of its kind.